orga/tests/integration/components/layout/sidebar_test.js

```diff
@@ -141,7 +141,7 @@ module('Integration | Component | Layout::Sidebar', function (hooks) {
       assert.dom('.sidebar-menu__documentation-item').doesNotExist();
     });
 
-    test('it should display team for a pro organization', async function (assert) {
+    test("it should display team for organization's members", async function (assert) {
```

Comment:
`'it should display an organization members\' team'`

Reply:
ça affiche la liste des membres pour tous les membres de cette organisation, tu dirais ça comment en anglais ?
sinon prettier me vire le \

Reply:
it should display the team for all organisation members

Reply:
merci ^^